specification.scs

```diff
@@ -0,0 +1,26 @@
+format_translators_ps_specification
+    <- concept_reusable_component_specification;;
+
+format_translators_ps_specification = [*
+-> rrel_key_sc_element : format_translators_ps
+(*
+    <- concept_reusable_component;;
+    <- concept_atomic_reusable_component;;
```

Comment:
```suggestion
    <- concept_non_atomic_reusable_component;;
```

Reply:
there is no subcomponents for that component so it is atomic